subjects/zappy/README.md

```diff
@@ -0,0 +1,351 @@
+
+ ----- ----- ----- ----- ----- ----- -----
+|  9  | 10  | 11  | 12  | 13  | 14  | 15  |
+ ----- ----- ----- ----- ----- ----- -----
+      |  4  |  5  |  6  |  7  |  8  |
+       ----- ----- ----- ----- -----
+             | 1  | 2  | 3  |
+              ---- ---- ----
+                  | 0  |
+                   ----
+                  Level 3
+
+```
+
+For the player to know what's around him, the `client` sends the command `SEE` and the server responds the following chain of characters (for level 1):
+
+```console
+{content-square-0, content-square-1, content-square-2, content-square-3}
+```
+
+Our player cannot see himself and if there is more than 1 object in a square, they are all indicated and separated by a space:
+
+```console
+              -------- -------- --------
+             | Amber  | Garnet |        |
+             | player | Garnet |        |
+              -------- -------- --------
+                      |        |
+                      |  food  |
+                       --------
+                        Level 1
+
+```
+
+```console
+{food, player Amber, Garnet Garnet, }
+```
+
+### The Time and Game Speed
+
+The game speed is defined by a time unit. Each action in the game has a duration proportionate to this time unit.
+
+The time unit is defined by the function 1/t :
+
+- t is passed as parameter at the start of the server.
+- A time unit lasts 1/t seconds.
+
+- There cannot be any blocking when the clients are stopped, or in any phase of the game.
+- The Minerthians have second as an absolute time unit.
+- `t` is an integer. If `t=1` "advance" takes 7 seconds. We choose by default, `t=100` then "advance" will take 1/100 seconds.
+
+### Objects denomination
+
+It is impossible to distinguish two objects of the same class, only the classification of an object can be identifiable,
+that is, two 'ambers' will have the same denomination, as they belong to the same class.
+
+### Teams
+
+At the beginning a team is made of `n` player and only `n`. Each player is controlled by a client. The clients cannot communicate or exchange amongst each other data outside of the game, in any way.
+
+At the beginning the client has 10 food units, he can therefore survive 1260 time units, 1260/t seconds.
+
+### Reproduction
+
+Executing the `fork` command results in the production of an `egg`, which allows for reproduction.
+
+Once the egg is laid, the player who laid the egg can go to it and hatch it, when it hatches, a new player will be born and given a random direction.
+This operation authorizes the connection of a new client.
+
+The `connect_nbr` command returns the number of authorized and unauthorized connections for this team.
+
+- Time to lay the egg: 48/t
+
+- Time for the egg to hatch 600/t
+
+### Inventory
+
+The `inventory` command allows you to see what objects the player has and how much time he has left. The server will send back, for example, the following line:
+
+```
+{food 300, Amber 4, Garnet 7, ..., Peridot 2}
```

Comment:
same issue of mixed caps, maybe all lower will be better

Reply:
done